pkg/ddl/ddl.go

```diff
@@ -93,6 +93,20 @@ var (
 	detectJobVerInterval = 10 * time.Second
 )
 
+// StartMode is an enum type for the start mode of the DDL.
+type StartMode string
+
+const (
+	// Normal mode, cluster is in normal state.
+	Normal StartMode = "normal"
```

Comment:
What is difference between normal mode with bootstrap mode? I mean whatever update or normal situation, we both need bootstrap.